src/_guides/language/analysis-options.md

```diff
@@ -204,6 +204,50 @@ For example, some rules are more appropriate for library packages
 and others are designed for Flutter apps.
 Note that linter rules can have false positives, unlike static analysis.
 
+### Enabling Dart team recommended rules: lints {#lints}
+
+The Dart team provides two sets of recommended linter rules.
+
+Two sets of lints are provided by the package:
+
+Core lints
+: Lints that help identify critical issues that are likely to lead to problems
+when running or consuming Dart code. All code should pass these lints.
+
+Recommended lints
+: Lints that help identify additional issues
+that may lead to problems when running or consuming Dart code,
+and lints that enforce writing Dart using a single, idiomatic style and format.
+All code is encouraged to pass these lints.
+The recommended lints include all the core lints.
+
+To enable one the Dart team's recommended sets of lints,
+add the [lints package]({{site.pub-pkg}}/lints) as a dev dependency:
+
+```terminal
+$ dart pub add --dev lints
+```
+
+Then include the `analysis_options.yaml` file
+corresponding to your preferred set
+in your `analysis_options.yaml` file:
```

Comment:
It's usually a bad sign to have two code blocks on top of each other. And also, these files aren't named `analysis_options.yaml`. So maybe change this paragraph like so:

```suggestion
Then edit your `analysis_options.yaml` file to include
the analysis options file for your preferred rule set:
```